src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_ActiveFareProductsTabScreen.tsx

```diff
@@ -32,7 +34,13 @@ export const TicketTabNav_ActiveFareProductsTabScreen = () => {
         reservations={reservations}
         fareContracts={activeFareContracts}
         isRefreshing={isRefreshingFareContracts}
-        refresh={resubscribeFirestoreListeners}
+        refresh={() => {
+          resubscribeFirestoreListeners();
+          analytics.logEvent('Ticketing', 'Pull to refresh tickets', {
```

Comment:
Is this ment to log the counts before or after the resubscribing was done? If it is ment to log the counts before, I think it is more explicit to put the event before the resubscribe function invocation. If it is ment to be the counts after, I am not sure they have had the time to be updated after the resubscription.